app/src/unitTests/java/com/artemzin/qualitymatters/models/PicassoBitmapClientTest.java

```diff
@@ -0,0 +1,56 @@
+package com.artemzin.qualitymatters.models;
+
+import android.support.annotation.NonNull;
+import android.widget.ImageView;
+
+import com.artemzin.qualitymatters.QualityMattersRobolectricUnitTestRunner;
+import com.squareup.picasso.Picasso;
+import com.squareup.picasso.RequestCreator;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+@RunWith(QualityMattersRobolectricUnitTestRunner.class)
+public class PicassoBitmapClientTest {
+
+    @SuppressWarnings("NullableProblems") // Initialized in @Before.
+    @NonNull
```

Comment:
@vanniktech I agree with @artem-zinnatullin. With test classes you may do the instantiation of class attributes through a mocking library (mockito) and/or it might take place far away from the definition spatially in the source code. @NonNull support annotations are just good visual feedback to remind the reader that it is instantiated elsewhere.